tools/bin2cpp/bin2cpp.cpp

```diff
@@ -105,8 +105,8 @@ int main(int argc, char* argv[])
 
 	strcpy(srcfile,argv[ARG_SRCFILE]);
 
-	int srcfn_len = strlen( srcfile );
-	if( srcfile[srcfn_len-4] != '.' )
+	const size_t srcfn_len = strlen( srcfile );
+	if( srcfn_len < 0 || srcfile[srcfn_len-4] != '.' )
```

Comment:
size_t is unsigned, checking if less than 0

Reply:
Whoops, that was meant to be a 4 (to avoid the buffer underflow).